x-pack/plugins/session_view/public/components/detail_panel_process_tab/styles.ts

```diff
@@ -30,10 +22,15 @@ export const useStyles = () => {
       paddingLeft: euiTheme.size.xs,
     };
 
+    const ellipsis: CSSObject = {
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
```

Comment:
don't need to add `whiteSpace: 'nowrap',` as well?

Reply:
Yea seems like in this case having `whiteSpace: 'nowrap'` on the parent level (tooltip anchor) was enough 
<img width="726" alt="image" src="https://user-images.githubusercontent.com/16872649/165124881-8f8ed521-be56-417f-9d38-0110a04c4581.png">